README.md

```diff
@@ -72,3 +72,11 @@ GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
 ```
 
 If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+
+### Typos Check
+
+You can use [spell checker](https://github.com/crate-ci/typos) to find and correct spelling mistakes after you updated the documents.
+
+To install the spell checker tool, you can refer to [Install](https://github.com/crate-ci/typos?tab=readme-ov-file#install).
+
+Then, just executing `typos . --config ./typos.toml` at the repo root path in your local commandline.
```

Comment:
```suggestion

```
It shouldn't be present in README.

Reply:
Why not? The previous section describes how to contribute documents to this repository and why can not include a spell checking step after written documents?

Reply:
This content is more like the troubleshooting things, it should be present at https://karmada.io/docs/contributor/contribute-docs#debugging-docs.

Reply:
done